### PR TITLE
ceph-grafana-trigger: update grafana container image version to 8.2.6

### PR DESCRIPTION
**Pull request by aaSharma14**
update grafana container image version to 8.2.6

Signed-off-by: Aashish Sharma <aasharma@redhat.com>

### CODE REVIEW

File: ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@
       - string:
           name: CONTAINER_VERSION
           description: "The version tag for the containers; will have -ARCH added to it.  By convention, the version of Grafana"
-          default: "6.7.4"
```
Comment (epuertat) [inline ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml:26]: Just curious: any specific reason for 8.2.6 and not 8.4.2 or 8.1.2?

**Review by epuertat — APPROVED** [review]
LGTM as far as there were were no regressions testing with the existing Dashboards.

BTW, any chance to add a trigger so that this is automatically run:

image

File: ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml (hunk 1 above)
Comment (aaSharma14) [reply ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml:26]: I was checking this out - https://hub.docker.com/r/grafana/grafana and the latest stable version i found here is 8.2.6, so decided to go forward with this.

Comment (epuertat) [reply ceph-grafana-trigger/config/definitions/ceph-grafana-trigger.yml:26]: Makes sense!

**Comment by epuertat** [comment]
Build launched: https://jenkins.ceph.com/view/all/job/ceph-grafana-trigger/2/

**Pull request merged**